app/views/assignments/_cp_assignment_upload_modal.html.erb

```diff
@@ -0,0 +1,30 @@
+<%= content_for :modal_id, 'upload_dialog_config' %>
+<%= content_for :modal_title, t('upload_the', item: 'Configuration Zip') %>
+<%= content_for :modal_open_link, '#uploadModalConfig' %>
+
+<%= content_for :modal_content do %>
+
+  <!-- Insert assignment-configuration section_id once documentation has been created -->
+  <%= t('assignments.upload_config_help_html', section_id: '') %>
```

Comment:
I would just add the section_id argument here based on your proposed wiki changes ('assignment-configuration'). That way you don't have to remember to come back to this and fix it later on.

As a side note... usually it's good practice to add `TODO: ` at the beginning of these sort of comments. That makes it easier to find them later when you're doing a final cleanup to make sure you didn't forget anything.

Reply:
I was concerned this may lead to a broken link but I see your point. I will add it in. I will also keep the TODO in mind

Reply:
It's a good concern but the nice thing is in this case if the section_id is wrong it will still take you to the top of the file. Anything after the '#' in a url will take you to the element with that id but if it can't find it, it will just stay at the top of the file. So something like this just takes you to the top of the page: https://github.com/MarkUsProject/Markus/wiki/Instructor-Guide--Importing-and-Exporting-Data#this-is-not-an-actual-secion